src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/NoOpRestoreUtilities.cs

```diff
@@ -11,11 +11,13 @@
 using NuGet.Packaging.Core;
 using NuGet.ProjectModel;
 using NuGet.Protocol;
+using NuGet.Versioning;
 
 namespace NuGet.Commands
 {
     public class NoOpRestoreUtilities
     {
+        internal static readonly string NoOpCacheFileName = "project.assets.nuget.cache";
```

Comment:
project.nuget.cache maybe?
The current template is projectName.projectExt.nuget.cache, so that seems like a natural change.